dist/index.js

```diff
@@ -50,6 +74,10 @@ var _either = _interopRequireDefault(require("./either.js"));
 
 var _formatting = require("./formatting.js");
 
+var _formatting2 = require("./formatting");
```

Comment:
Can we get rid off one?

Reply:
One line is generated from an `import` and the other from an `export ... from`. I think the source reads clearer this way (the `import` is a dependency for the module while the `export` is for the purpose stated in this PR) and there is no impact in having two lines in the `dist` like this